fp: [0.8, 0.7, 0.7]
fi: [0.05, 0.04, 0.01]
fd: [0.0, 0.0, 0.0]

mp: [0.2, 0.1, 0.15]
mi: [0.01, 0.01, 0.0]
md: [0.0, 0.0, 0.0]

imp_d_: 7.5 #30.0 #31.6228
imp_k_: 12.5 #50.0 #50.
imp_scale_: 5.0

imp_m_: 4.9 #3.0

imp_f_: 0.1

imp_scale_m_: 0.0 #0.5 #rigid0.5 soft?
imp_scale_m_i_: 0.0 #0.05 # rigid 0.05 soft ?